controllers/api/imap.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	ctx "github.com/gophish/gophish/context"
+	"github.com/gophish/gophish/imap"
+	log "github.com/gophish/gophish/logger"
+	"github.com/gophish/gophish/models"
+)
+
+// IMAPServerValidate handles requests for the /api/imapserver/validate endpoint
+func (as *Server) IMAPServerValidate(w http.ResponseWriter, r *http.Request) {
+	switch {
+	case r.Method == "GET":
+		JSONResponse(w, models.Response{Success: false, Message: "Only POSTs allowed"}, http.StatusBadRequest)
+	case r.Method == "POST":
+		im := models.IMAP{}
+		// Put the request into a page
```

Comment:
Small nit - this comment looks like it came from the `Pages` handler.

Reply:
Removed